handbook/python-dataplugins/python-dataplugins.md

```diff
@@ -0,0 +1,308 @@
+# Python DataPlugins
+
+Create a DataPlugin to load, to register, or to search your own file formats in LabVIEW or DIAdem, or to index and to browse your own file formats with SystemLink DataFinder.
+
+You can create DataPlugins using C++, VBS, LabVIEW or Python.
+
+## Getting Started
+
+Python DataPlugins consist of only one Python file that contains all the logic. Almost all language features of the official Python 3.5.9 and its base libraries can be used.
+
+You can create Python DataPlugins in your favorite editor. Recommended:
+
+- [DIAdem](https://www.ni.com/en-us/shop/data-acquisition-and-control/application-software-for-data-acquisition-and-control-category/what-is-diadem.html) >= 2020
+- VSCode with the [NI Python DataPlugin Extension](https://github.com/ni/vscode-ni-python-dataplugins)
+
+### Plugin class
+
+Start writing your DataPlugin by implementing
+
+```python
+class Plugin:
+```
+
+The class name cannot be changed.
+
+### Store read
+
+Every Python DataPlugin needs to implement a `read_store` method. This method is called by DIAdem, LabVIEW or SystemLink DataFinder when it attempts to open your data file. The applications pass a set of useful parameters that can be accessed by the parameter array.

```

Comment:
Unsure here: Does the Python DataPlugin implement a "read_store" method, or does the user implement the "read_store" method? Or does implement mean "execute" in this case? Sorry, I'm not a programmer...
Maybe: "Every Python DataPlugin needs a "read_store" method." or "You need to implement a "read_store" method in every DataPlugin.

This method is called by DIAdem, LabVIEW, or SystemLink DataFinder when attempting to open your data file.

Reply:
> "You need to implement a "read_store" method in every DataPlugin."
thanks, this is good.